src/main/docker/gitlab/gitlab-local-setup.sh

```diff
@@ -0,0 +1,55 @@
+
+#!/bin/bash
+
+# Enter the personal access token of the admin user
+ADMIN_PERSONAL_ACCESS_TOKEN="your.gitlab.admin.access.token"
+ADMIN_USER_ID="1"
+
+# Enter the url of the Gitlab instance.
+GITLAB__API_URL="http://localhost/api/v4/"
```

Comment:
Are the two underscores in GITLAB__API_URL that what you want? I think that could lead to errors.